modules/ROOT/pages/functions/string.adoc

```diff
@@ -148,6 +148,163 @@ RETURN ltrim('   hello')
 
 ======
 
+
+
+[[functions-normalize]]
+== normalize()
+
+`normalize()` returns the given `STRING` normalized using the `NFC` normalization form.
```

Comment:
Added a note to make clear to readers (like me) who are not sure what a normalization form is.

```suggestion
`normalize()` returns the given `STRING` normalized using the `NFC` Unicode normalization form.

[NOTE]
====
Unicode normalization is a process that transforms different representations of the same text into a standardized form.
For more information, see the documentation for link:https://unicode.org/reports/tr15/#Norm_Forms[Unicode normalization forms].
====
```